tests/pc/test_lag_2.py

```diff
@@ -368,47 +368,47 @@ def teardown(duthost):
     return
 
 
-def get_oper_status_from_db(duthost, port_name):
+def get_oper_status_from_db(asichost, port_name):
     """Get netdev_oper_status from state_db for interface"""
-    cmd = "redis-cli -n 6 hget \"PORT_TABLE|{}\" netdev_oper_status".format(port_name)
-    status = duthost.shell(cmd, module_ignore_errors=False)['stdout']
+    cmd = asichost.get_docker_cmd("redis-cli -n 6 hget \"PORT_TABLE|{}\" netdev_oper_status".format(port_name), "database")
+    status = asichost.sonichost.shell(cmd, module_ignore_errors=False)['stdout']
     # If PORT_TABLE in STATE_DB doesn't have key netdev_oper_status,
     # check oper_status in APPL_DB instead. This scenario happens on 202012.
     if not status:
-        cmd = "redis-cli -n 0 hget \"PORT_TABLE:{}\" oper_status".format(port_name)
-        status = duthost.shell(cmd, module_ignore_errors=False)['stdout']
+        cmd = asichost.get_docker_cmd("redis-cli -n 0 hget \"PORT_TABLE:{}\" oper_status".format(port_name), "database")
+        status = asichost.sonichost.shell(cmd, module_ignore_errors=False)['stdout']
     return status
 
-def get_admin_status_from_db(duthost, port_name):
+def get_admin_status_from_db(asichost, port_name):
     """Get netdev_oper_status from state_db for interface"""
-    cmd = "redis-cli -n 6 hget \"PORT_TABLE|{}\" admin_status".format(port_name)
-    status = duthost.shell(cmd, module_ignore_errors=False)['stdout']
+    cmd = asichost.get_docker_cmd("redis-cli -n 6 hget \"PORT_TABLE|{}\" admin_status".format(port_name), "database")
```

Comment:
I'm not sure about this hget, on another multi-asic dut it is not recognized:

admin@masic-acs-1:~$ sudo docker exec database1 redis-cli -n 6 hget "PORT_TABLE|Ethernet28"
ERR wrong number of arguments for 'hget' command

admin@masic-acs-1:~$ redis-cli -n 6 hget "PORT_TABLE|Ethernet28"
(error) ERR wrong number of arguments for 'hget' command

Reply:
thanks. that makes sense